policy/comCamMapper.yaml

```diff
@@ -0,0 +1,430 @@
+defects: ../description/defects
+
+needCalibRegistry: true
+
+levels:
+  tract: patch
+
+defaultLevel: visit
+
+exposures:
+  _raw:
+    level: Ccd
+    persistable: PropertyList
+    python: lsst.daf.base.PropertyList
+    storage: FitsStorage
+    tables: raw
+    template: raw/%(run)s/%(raft)s/%(ccd)s/%(raft)s-%(ccd)s-%(visit)06d.fits[0]
+  raw:
+    assembler: lsst.obs.comCam.comCamMapper.assemble_raw
+    composite:
+      raw_amp:
+        datasetType: raw_amp
+        subset: true
+      raw_hdu:
+        datasetType: raw_hdu
+    inputOnly: true
+  raw_amp:
+    level: Amp
+    persistable: DecoratedImageI
+    python: lsst.afw.image.DecoratedImageI
+    storage: FitsStorage
+    tables: raw
+    template: raw/%(run)s/%(raft)s/%(ccd)s/%(raft)s-%(ccd)s-%(visit)06d.fits[%(channel)d]
+  postISRCCD:
+    level: Ccd
+    persistable: ExposureF
+    python: lsst.afw.image.ExposureF
+    storage: FitsStorage
+    tables: raw
+    template: postISRCCD/postISRCCD_v%(visit)d_f%(filter)s.fits
+  calexp:
+    level: Ccd
+    persistable: ExposureF
+    python: lsst.afw.image.ExposureF
+    storage: FitsStorage
+    tables: raw
+    template: calexp/calexp_v%(visit)d_f%(filter)s.fits
```

Comment:
obs_base defines most of these data products, sometimes with fields missing that individual obs_ packages must then define -- typically just the template. I strongly suggest taking a cleanup pass to remove all duplicate information.